xamarin/Xamarin.Forms
# [iOS] Fix Path TranslateTransform issue

### Description of Change ###

 Fix Path TranslateTransform issue on iOS. The issue was applying the transformation incorrectly.

### Issues Resolved ### 

- fixes #11113 

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix-11113](https://user-images.githubusercontent.com/6755973/85112787-7d557a80-b216-11ea-9d68-05e9b7dfaef8.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue **11113**. Move the Translation Sliders and verify that the Path moves.

### PR Checklist ###
